lib/QueueWindowProcessor.ts

```diff
@@ -0,0 +1,24 @@
+import { IHttp, IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { IJobContext, IProcessor } from '@rocket.chat/apps-engine/definition/scheduler';
+import { Logs } from '../enum/Logs';
+import { getError } from './Helper';
+import { setIsQueueWindowActive} from './Persistence';
+
+export class QueueWindowScheduler implements IProcessor {
+    public id: string;
+
+    constructor(id: string) {
+        this.id = id;
+    }
+
+    public async processor(jobContext: IJobContext, read: IRead, modify: IModify, http: IHttp, persistence: IPersistence): Promise<void> {
+        const sessionId = jobContext.rid;
+        try {
+            await setIsQueueWindowActive(read, persistence, sessionId, false);
+        } catch (error) {
+            // Failed to send event, so close blackout window
+            console.error(`${Logs.DIALOGFLOW_REST_API_ERROR}: { roomID: ${sessionId} } ${getError(error)}`);
+        }
+
+    }
```

Comment:
You are trying to set`setIsQueueWindowActive` flag to `false` But the actual flag that blocks user messages is `setIsProcessingMessage` So we need to use that maybe?